internal/accounts/contract-add.go

```diff
@@ -57,14 +59,34 @@ func addContract(
 	srv *services.Services,
 	state *flowkit.State,
 ) (command.Result, error) {
-	name := args[0]
-	filename := args[1]
+	name := ""
+	filename := ""
+
+	if len(args) == 1 {
+		filename = args[0]
+	} else {
+		filename = args[1]
+	}
 
 	code, err := readerWriter.ReadFile(filename)
 	if err != nil {
 		return nil, fmt.Errorf("error loading contract file: %w", err)
 	}
 
+	resolver, err := contracts.NewResolver(code)
```

Comment:
This should be done inside the account service.  Resolver is already created there https://github.com/onflow/flow-cli/blob/master/pkg/flowkit/services/accounts.go#L309